lib/src/user_event/user_event.dart

```diff
@@ -63,6 +63,13 @@ abstract class UserEvent {
   /// [UIEvent.detail](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/detail)
   /// for more information.
   ///
+  /// ### [skipPointerEventsCheck]
+  ///
+  /// Use [skipPointerEventsCheck] to skip checking if any element
+  /// in the DOM-tree has `'pointer-events: none'` set. This check is
```

Comment:
Maybe could also add that `click` will usually throw an error if `'pointer-events: none'` is set - so this is changing that